pom.xml

```diff
@@ -85,6 +85,15 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.2</version>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
I was wondering if there is the posibility to inherit the used jdk from the environment. This would allow us to check the build against other jdk´s as well. For example in travis:
`.travis.yml`
```yml
language: java
jdk:
  - oraclejdk8
  - oraclejdk7
  - oraclejdk6
[...] 
```